Lib9c/Model/Quest/QuestReward.cs

```diff
@@ -14,7 +14,10 @@ public class QuestReward : IState
 
         public QuestReward(Dictionary<int, int> map)
         {
-            ItemMap = map;
+            ItemMap = map
+                .OrderBy(kv => kv.Key)
```

Comment:
I guess this `.OrderBy()` doesn't affect result because we convert it through `.ToDictionary()` directly.